### PR TITLE
Update chooseyourday to latest sdk

### PR DESCRIPTION
**Pull request by jenow**

### CODE REVIEW

File: chooseyourday/bower.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Kuzzle arrow hero demo",
```
Comment (dbengsch) [inline chooseyourday/bower.json]: Choose your day :)

Comment (j33f) [inline chooseyourday/bower.json]: :+1: :laughing:

File: arrow-hero/js/classes/kuzzle-manager.js [hunk 2]
```diff
@@ -76,10 +82,10 @@ KuzzleGame.KuzzleManager = {
 
     KuzzleGame.KuzzleManager.log('registering as host');
 
-    this.kuzzle.create("kg_main_room", {
+    this.kuzzle.dataCollectionFactory("kg_main_room", this.mainIndex).createDocument({
```
Comment (stafyniaksacha) [inline arrow-hero/js/classes/kuzzle-manager.js]: Index and collection are in the inverse order

File: arrow-hero/js/classes/kuzzle-manager.js [hunk 3]
```diff
@@ -126,7 +132,7 @@ KuzzleGame.KuzzleManager = {
     if (typeof(clearHostId) === 'undefined') clearHostId = true;
 
 
-    this.kuzzle.deleteByQuery("kg_main_room", filters, function(error, response) {
+    this.kuzzle.dataCollectionFactory(this.mainRoom, this.mainIndex).deleteDocument(filters, (error, response) => {
```
Comment (stafyniaksacha) [inline arrow-hero/js/classes/kuzzle-manager.js]: Index and collection are in the inverse order

File: arrow-hero/js/classes/kuzzle-manager.js [hunk 4]
```diff
@@ -157,9 +163,9 @@ KuzzleGame.KuzzleManager = {
    */
   createHostSubChannel: function() {
     if (this.hostID) {
-      this.kuzzle.create("kg_room_" + this.hostID, {
+      this.kuzzle.dataCollectionFactory("kg_room_" + this.hostID, this.mainIndex).createDocument({
```
Comment (stafyniaksacha) [inline arrow-hero/js/classes/kuzzle-manager.js]: Index and collection are in the inverse order

File: arrow-hero/js/classes/kuzzle-manager.js [hunk 5]
```diff
@@ -184,7 +190,7 @@ KuzzleGame.KuzzleManager = {
       }
     };
 
-    this.kuzzle.deleteByQuery("kg_room_" + this.hostID, filters, function(error, response) {
+    this.kuzzle.dataCollectionFactory("kg_room_" + this.hostID, this.mainIndex).deleteDocument(filters, (error, response) => {
```
Comment (stafyniaksacha) [inline arrow-hero/js/classes/kuzzle-manager.js]: Same as above

File: arrow-hero/js/classes/kuzzle-manager.js [hunk 6]
```diff
@@ -208,7 +214,7 @@ KuzzleGame.KuzzleManager = {
       }
     };
 
-    this.kuzzle.subscribe("kg_room_" + this.hostID, filters, this.fireEvent);
+    this.kuzzle.dataCollectionFactory("kg_room_" + this.hostID, this.mainIndex).subscribe(filters, this.fireEvent);
```
Comment (stafyniaksacha) [inline arrow-hero/js/classes/kuzzle-manager.js]: Same as above

File: arrow-hero/js/classes/kuzzle-manager.js [hunk 7]
```diff
@@ -246,12 +252,12 @@ KuzzleGame.KuzzleManager = {
    * @param value
    */
   throwEvent: function(eventType, value) {
-    this.kuzzle.create("kg_room_" + this.hostID, {
+    this.kuzzle.dataCollectionFactory("kg_room_" + this.hostID, this.mainIndex).createDocument({
```
Comment (stafyniaksacha) [inline arrow-hero/js/classes/kuzzle-manager.js]: Same as above

File: arrow-hero/js/classes/kuzzle-manager.js [hunk 8]
```diff
@@ -9,27 +9,34 @@ KuzzleGame.KuzzleManager = {
   connexionEstablished: false,
   connexionLastCheck: 0,
   connexionInterval: false,
-  debug: false,
+  debug: true,
```
Comment (stafyniaksacha) [inline arrow-hero/js/classes/kuzzle-manager.js]: Debug must be false

**Pull request merged**